src/Specific/spec.js

```diff
@@ -173,5 +165,17 @@ describe('Test `is` (specific) function', () => {
         it('Should return false for a boolean', () => {
             expect(Specific.is(true)).to.equal(false);
         });
+
+        it('Should return false for an Object without `not` or `only`', () => {
+            expect(Specific.is({})).to.equal(false);
+        });
+
+        it('Should return false for null', () => {
+            expect(Specific.is(null)).to.equal(false);
+        });
+
+        it('Should return false for undefined', () => {
+            expect(Specific.is(null)).to.equal(false);
+        });
```

Comment:
```js
it('Should return false for empty string', () => {
  expect(Specific.is('')).to.equal(false);
});

it('Should return false for empty array', () => {
  expect(Specific.is([])).to.equal(false);
});
```
With this test you can remove the line `if (specific.length === 0) { return; }` in the Specific constructor

Reply:
As I explained earlier, I don't think it should return false.